sky/clouds/aws.py

```diff
@@ -544,3 +549,40 @@ def accelerator_in_region_or_zone(self,
                                       zone: Optional[str] = None) -> bool:
         return service_catalog.accelerator_in_region_or_zone(
             accelerator, acc_count, region, zone, 'aws')
+
+    @classmethod
+    def check_disk_type_enabled(
+        cls,
+        instance_type: str,  # pylint: disable=unused-argument
+        disk_type: str  # pylint: disable=unused-argument
+    ) -> None:
+        return
+
+    @classmethod
+    def get_disk_type(cls, disk_type: str) -> str:
+        # aws disk will be configured to different IOPS and throughput
+        return 'standard' if disk_type == 'low' else 'gp3'
+
+    @classmethod
+    def get_disk_iops(cls, disk_type: str) -> int:
+        type2iops = {
+            'high': 7000,
+            'medium': 3500,
+            'low': 0,  # only gp3 is required to set iops
+        }
+        return type2iops[disk_type]
+
+    @classmethod
+    def get_disk_throughput(cls, disk_type: str) -> int:
+        return cls.get_disk_iops(disk_type) // 16
+
+    @classmethod
+    def enable_custom_disk_perf(cls, disk_type: str) -> bool:
+        return cls.get_disk_type(disk_type) == 'gp3'
+
+    @classmethod
+    def get_disk_desc(cls, disk_type: str) -> str:
+        cloud_disk_type = cls.get_disk_type(disk_type)
+        cloud_disk_iops = cls.get_disk_iops(disk_type)
+        return f'{disk_type}:{cloud_disk_type}[{cloud_disk_iops}]' if cls.get_disk_type(
+            disk_type) == 'gp3' else f'{disk_type}:{cloud_disk_type}'
```

Comment:
Can we make these functions a single one `_get_disk_specs`, as they are only used once and we may want to avoid shallow functions?
```
def _get_disk_specs(...) -> Dict[str, Any]:
    ...
```